src/Xive/Comb/RamComb.cs

```diff
@@ -98,9 +100,38 @@ public IXocument Xocument(string name)
             return this.xocumentWrap(new CellXocument(Cell(name), name));
         }
 
-        public ICell Cell(string name)
+        public ICell Cell(string cell)
         {
-            return this.cellWrap(new RamCell($"{this.name}{Path.DirectorySeparatorChar}{name}", cellMemory));
+            ICell result;
+            if (cell.Equals("_guts.xml"))
+            {
+                var patch = new Directives().Add("items");
+
+                foreach (var name in this.cellMemory.Keys)
+                {
+                    patch.Add("item")
+                        .Add("name")
+                        .Set(name) //file.Substring((this.name + "/").Length))
```

Comment:
We want guts to tell us the path from here - the comb. So you must take the substring, cutting off the comb name from the full cell name.

The dictionary will alwas contain a full name, think of it like a file path. Same concept.